src/nvim/api/ui.c

```diff
@@ -314,13 +318,17 @@ void nvim_ui_try_resize_grid(uint64_t channel_id, Integer grid, Integer width,
   ui_grid_resize((handle_T)grid, (int)width, (int)height, err);
 }
 
-/// Tells Nvim the number of elements displaying in the popumenu, to decide
-/// <PageUp> and <PageDown> movement.
+/// Tells Nvim the geometry of the popumenu, to decide <PageUp> 
+/// and <PageDown> movement, and to align floating windows with
+/// an external popup menu.
 ///
 /// @param channel_id
+/// @param width   Popupmenu width, must be greater than zero.
 /// @param height  Popupmenu height, must be greater than zero.
-/// @param[out] err Error details, if any
-void nvim_ui_pum_set_height(uint64_t channel_id, Integer height, Error *err)
```

Comment:
No, this is not the same thing. This method sets the _number of items_ visible in the pum, while the new method would set the physical dimension of the pum. If the font size/line spacing is not the same (or the pum has a border), then these numbers will not be the same.

Reply:
understood.

Reply:
updated. also a little bit docs on this.